core/wire/kem.go

```diff
@@ -295,59 +295,86 @@ type scheme struct {
 
 var _ Scheme = (*scheme)(nil)
 
+func (s *scheme) PrivateKeyFromBytes(b []byte) (PrivateKey, error) {
+	privKey, err := s.KEM.ParsePrivateKey(b)
+	if err != nil {
+		return nil, err
+	}
+	return &privateKey{
+		privateKey: privKey,
+		KEM:        s.KEM,
+	}, nil
+}
+
+func (s *scheme) PublicKeyFromBytes(b []byte) (PublicKey, error) {
+	pubKey, err := s.KEM.ParsePublicKey(b)
+	if err != nil {
+		return nil, err
+	}
+	return &publicKey{
+		publicKey: pubKey,
+		KEM:       s.KEM,
+	}, nil
+}
+
 func (s *scheme) PrivateKeyFromPemFile(f string) (PrivateKey, error) {
-	privKey := s.GenerateKeypair(rand.Reader)
-	err := cpem.FromFile(f, privKey)
+	keyType := fmt.Sprintf("%s PRIVATE KEY", strings.ToUpper(s.KEM.String()))
+	buf, err := os.ReadFile(f)
 	if err != nil {
 		return nil, err
 	}
-	return privKey, nil
+	blk, _ := pem.Decode(buf)
+	if blk == nil {
+		return nil, fmt.Errorf("failed to decode PEM file %v", f)
+	}
+	if blk.Type != keyType {
```

Comment:
this should be `strings.ToUpper(blk.Type)` because the keytype is `Kyber768-X25519` in a bunch of places and `KYBER768-X25519` in other places, so this fails to read e.g. the keys in `client/testdata/client.toml`

Reply:
same issue for the public keys I'm guessing?

Reply:
done